packages/wrangler/src/pages/download-config.ts

```diff
@@ -62,29 +69,49 @@ export interface PagesProject extends Project {
 }
 
 async function toEnvironment(
-	project: PagesDeploymentConfig,
+	deploymentConfig: PagesDeploymentConfig,
 	accountId: string
 ): Promise<RawEnvironment> {
 	const configObj = {} as RawEnvironment;
 	configObj.compatibility_date =
-		project.compatibility_date ?? new Date().toISOString().substring(0, 10);
-	if (project.compatibility_flags?.length)
-		configObj.compatibility_flags = project.compatibility_flags;
+		deploymentConfig.compatibility_date ??
+		new Date().toISOString().substring(0, 10);
+
+	// Find the latest supported compatibility date and use that
+	if (deploymentConfig.always_use_latest_compatibility_date) {
```

Comment:
I don't see these warnings. I think you needed to update your dependencies @CarmenPopoviciu  to see these, since this type is extracted from the `@cloudflare/types` library.

Reply:
yes, this was my editor settings playing out. I'll resolve the conversation